backend/src/news/utils.py

```diff
@@ -41,27 +41,11 @@ def fetch_latest_news(is_initial=False):
     news_data = fetch_latest_news_info("價格", is_initial=is_initial)
     for news in news_data:
         title = news.title
-        relevance_judge_prompt = [
-            {
-                "role": "system",
-                "content": "你是一個關聯度評估機器人，請評估新聞標題是否與「民生用品的價格變化」相關，並給予'high'、'medium'、'low'評價。(僅需回答'high'、'medium'、'low'三個詞之一)",
-            },
-            {"role": "users", "content": f"{title}"},
-        ]
-        relevance = llm_generate(relevance_judge_prompt)
-        if relevance == "high":
+        relevance = llm_client.evaluate_relevance(title, "民生用品的價格變化")
+        if relevance == RelevanceEvaluation.high:
             detailed_news = udn_crawler.validate_and_parse(news.url)
 
-            summarizer_prompt = [
-                {
-                    "role": "system",
-                    "content": "你是一個新聞摘要生成機器人，請統整新聞中提及的影響及主要原因 (影響、原因各50個字，請以json格式回答 {'影響': '...', '原因': '...'})",
-                },
-                {"role": "users", "content": " ".join(detailed_news["content"])},
-            ]
-
-            result = llm_generate(summarizer_prompt)
-            result = json.loads(result)
+            result = llm_client.generate_summary(" ".join(detailed_news["content"]))
```

Comment:
The `detailed_news` is a `News` object returned by `udn_crawler.validate_and_parse(news.url)` at line 46, and `News` has the `content` attribute whose type is not `list[str]` but `str`. As a result, the line 48 should be changed like this:

```suggestion
            result = llm_client.generate_summary(detailed_news.content)
```